genoml/preprocessing/munging.py

```diff
@@ -172,13 +135,44 @@ def plink_inputs(self):
             f"{self.output_datafile}"
         )
 
+        self.merged = merged
```

Comment:
Does `self.merged` get used anywhere else, or this is just defensive/for debugging purposes?

Reply:
It was set in the original, so I left it in. When I return it, it is a memoryview (read: not copy) of the `self.merged` frame. I do not imagine it is used but yeah